sdk/core/azure-core-http-netty/pom.xml

```diff
@@ -78,6 +78,23 @@
       <artifactId>netty-codec-http</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-http2</artifactId>
+      <version>4.1.42.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-unix-common</artifactId>
+      <version>4.1.42.Final</version>
```

Comment:
LGTM, one question, so we no longer inherits version from the parent?

Reply:
@JimSuplizio is working on removing parent pom completely in this PR - https://github.com/Azure/azure-sdk-for-java/pull/6434

So, all versions will be explicitly added to every dependency in every pom file.

Reply:
https://mvnrepository.com/artifact/io.netty/netty-transport-native-unix-common/4.1.43.Final
Could we go to more latest version ?